src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operations.OperationHandler;
+
+public interface FruitShopService {
+    void addToStorage(FruitTransaction fruitTransaction, Storage storage,
```

Comment:
It's better to use more generic name (`process` for example) as your service is not only adding.